README

```diff
@@ -73,12 +73,123 @@ from using single-DES cryptosystems.  Among these is a configuration
 variable that enables "weak" enctypes, which defaults to "false"
 beginning with krb5-1.8.
 
-Major changes in 1.14
+Major changes in 1.15
 ---------------------
 
-krb5-1.14 changes by ticket ID
+Administrator experience:
+
+* Add support to kadmin for remote extraction of current keys without
+  changing them (requires a special kadmin permission), with the
+  exception of highly protected keys.
```

Comment:
I'd split this out into two items, since the lockdown_keys attribute fulfills a larger purpose than just disabling the new extraction feature.  For the second one, I suggest "Add a lockdown_keys principal attribute to prevent retrieval of the principal's keys (old or new) via the kadmin protocol.  In new databases, this attribute is set on the krbtgt and kadmin principals."